src/gas-hub.js

```diff
@@ -398,6 +418,44 @@ function updateBranch() {
   });
 }
 
+function handleRepoCreated(repo) {
+  return scm.getRepos()
+  .then(updateRepo)
+  .then(updateBranch)
+  .then(() => {
+    $('#new-repo-name').val('');
+    $('#new-repo-desc').val('');
+    $('#new-repo-type').val('public');
+    showAlert(`Successfully create new repository ${repo}`);
+  })
+  .catch(() => {
+    throw new Error('Repository created, but failed to show the new repository.');
+  });
+}
+
+function handleBranchCreated(branch) {
+  return updateBranch()
+  .then(() => {
+    $('#new-branch-name').val('');
+    showAlert(`Successfully create new branch: ${branch}`);
+  })
+  .catch(() => {
+    throw new Error('Branch created, but failed to show the new branch.');
+  });
+}
+
+function handleGistCreated() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/79a1121575f894385b70c00ba30d867da9fde2de/src/gas-hub.js#L447"></span>‘handleGistCreated’ is defined but never used.</td>
    </tr>
  </tbody>
</table>